### PR TITLE
Update Podcast player Block CSS attr to use `jetpack` namespace

### PR DESCRIPTION
**Pull request by getdave**
Currently all CSS classes (and some other SVG attributes) used on the Block do not include a "namespace" which ties them to Jetpack. This makes them vulnerable to be overridden by / clashing with other Blocks by 3rd party vendors.

#### Changes proposed in this Pull Request:

This PR updates so that all CSS classes and SVG attributes contain the `jetpack-` prefix.

eg:

```css
.podcast-player__episode {}
``` 

becomes...

```css
.jetpack-podcast-player__episode {}
``` 

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
It adds (updates) features to an existing part of Jetpack.

##### On Master

* Add the Podcast player Block - note the layout and styling of the Block on both Editor and Frontend (please take screenshots for your reference).
* Inspect the DOM - note:
  - CSS classes  do not contain `jetpack-` prefix (eg: `.podcast-player__episode`.
  - SVG attributes do not contain `jetpack-` prefix.

##### On this branch

* Add the Podcast player Block - compare the layout and styling of the Block. It should be identical to that of `master`. Please use screenshots to compare.
* Inspect the DOM - note:
  - CSS classes  should now contain the `jetpack-` prefix (eg: `.jetpack-podcast-player__episode`.
  - SVG attributes should now contain `jetpack-` prefix.

#### Proposed changelog entry for your changes:

* Update all classnames and SVG attributes to use the `jetpack` prefix.

### CODE REVIEW

**Comment by marekhrabe** [comment]
`podcast-player.js` needs updating too, as it uses classes in its internal logic

**Comment by getdave** [comment]
> `podcast-player.js` needs updating too, as it uses classes in its internal logic

Fixed.

**Review by marekhrabe — DISMISSED** [review]
Works as expected now!

Keep an eye on #15072. If it gets shipped before this, you need to update the classes in newly created react components over there. If you ship before it, I'll update the react part.

**Comment by getdave** [comment]
> Works as expected now!

Thanks. Now waiting on Jetpack Crew.

**Comment by marekhrabe** [comment]
This will need an update that renames classes in the React part too. Since that was a huge PR, I'm happy to handle the rebase, feel free to ping me

**Comment by obenland** [comment]
@marekhrabe I was in the process of rebasing as you added your comment. remaining step is updating the classes in the React part

**Comment by obenland** [comment]
Unless I missed a class, this should be ready for another review

**Review by marekhrabe — DISMISSED** [review]

**Comment by getdave** [comment]
Needs a rebase 😢

**Review by jeherve — APPROVED** [review]
I've rebased, this works well for me. It should be good to merge once the tests pass.

**Comment by obenland** [comment]
@jeherve GitHub doesn't seem to like your review. It still says "Code owner review required"

**Comment by getdave** [comment]
Changes in this PR are now available on WordPress.com: r204793-wpcom

**Pull request merged**
